bin/e2e-ci

```diff
@@ -1,4 +1,4 @@
 #!/bin/bash
 # As discussed in the engineering roundtable 2021/01/25, we will revisit cypress-rails knapsack pro integration
 # for re-enabling parallelization of e2e tests later.
-RAILS_ENV=test E2E=true bundle exec rake cypress:run
\ No newline at end of file
+RAILS_ENV=test E2E=true bundle exec rake search:setup cypress:run
```

Comment:
Should `search:setup` always precede `cypress:run`? If so, would it make sense as a dependency?